Riot/Modules/directory/role/detail/RoleDetailViewController.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  RoleDetailViewController.swift
+//  Riot
+//
+//  Created by Naurin Afrin on 3/8/20.
+//  Copyright © 2020 matrix.org. All rights reserved.
+//
+
+import UIKit
+
+private enum SectionType: Int, CaseIterable {
+    case ROLE_PROFILE = 0
+    case ROLE_TITLE = 1
+    case SPECIALITY = 2
+    case LOCATION = 3
+    case TEAM = 4
+    case ORGANISATION_UNIT = 5
+    case PRACTITIONER_ROLE = 6
+    
+    var cellIdentifier: String {
+        switch self {
+        case .ROLE_PROFILE:
+            return "RoleProfileTableViewCell"
+        case .ROLE_TITLE:
+            return "RoleTitleTableViewCell"
+        case .SPECIALITY:
+            return "SpecialityTableViewCell"
```

Comment:
Suggest to remove lines 27, 29 and 31, so it is clearer that these all return "SpecialityTableViewCell"

Reply:
These are specified as they are because Swift switch statements need to be exhaustive or use the `default` switch case.  Given we may wish to customise the cells used for location, team and organisation_unit in the future I would recommend leaving the switch as exhaustive as per current configuration.